decoders/connector/netvox/r718n125/v1.0.0/payload.ts

```diff
@@ -0,0 +1,85 @@
+function Decoder_r718n125(input, port) {
```

Comment:
Install the package `npm install -D @tago-io/sdk` and declare the variable payload.
```ts
import { DataToSend } from "@tago-io/sdk/lib/types";

declare let payload: DataToSend[];
```

Probably when you declare the payload variable you will get errors on line 82 and 83 replace it with:
```ts
    const buffer = Buffer.from(payload_r718n125?.value as string, "hex");
    const data = Decoder_r718n125(buffer, port_r718n125 as number);
```